interface/patient_file/summary/add_edit_issue.php

```diff
@@ -45,7 +45,7 @@
     }
 }
 
-$issue = $_REQUEST['issue'];
+$issue = filter_var($_REQUEST['issue'], FILTER_VALIDATE_INT);
```

Comment:
As discussed above, what vulnerability are you fixing here? Isn't $issue already properly escaped everywhere?

Reply:
Above not yet addressed.

Reply:
The problem is in line 365
![image](https://user-images.githubusercontent.com/17809866/30899277-ea1e69a8-a367-11e7-9808-3221553b3b76.png)
I can add 'attr' but as I wrote above this is not prevents injection because it in the js script.